fhir-server-webapp/src/main/liberty/config/configDropins/disabled/jvm.options

```diff
@@ -1,8 +1,9 @@
-# Set a fixed max heap size
--Xmx3072M
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2021
+# 
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
 
-# Turn on verbose classloading
-#-verbose:class
-
-# Turn on verbose garbage collection
-#-verbose:gc
+# https://www.eclipse.org/openj9/docs/xxinitialrampercentage/
+-XX:InitialRAMPercentage=50.00
```

Comment:
I think this could lead to a larger-than-needed initial heap size in cases where the memory limit is set fairly high.
For example, currently the alvearie/ibm-fhir-server helm chart defaults to a request of 1 Gi and a limit of 4Gi.
With this setting, I think that request will be way off which could lead to trouble.

I think I slightly prefer a fixed min heap size since we've observed idle memory usage in the 512 - 768 Mi range.

If we do go with the initialRAMPercent, I think we need to:
* document it at https://hub.docker.com/r/ibmcom/ibm-fhir-server
* change the helm chart so that the memory limit is roughly twice the memory request (probably dropping our limit from 4Gi to 2 Gi)

Reply:
One nice thing about using RAMPercentage in our defaults is that the `-Xms` and `-Xmx` options naturally override those.  So we certainly could have a default initialRAMPercent and then change the default to a fixed min heap size in the helm chart (if we wanted).

Reply:
Yes, that is possible.

Reply:
opted for the default.

Reply:
> opted for the default

I think that comment was intended for the max heap, right?  50% is definitely not the default on initial heap.
Personally, I think I'd still prefer a fixed initial heap size.  The default is 8 Mi.  Previously we went with 768 Mi.  I think it makes sense to go somewhere between there so that we can:
1. start in more constrained environments 
2. start in environments with lots of RAM available without oversizing the heap
3. avoid unnecessary resizing (although its still not clear to me how costly that actually is)

Reply:
I propose an alterative. Let's close the current PR/Issue as won't fix. and update the documentation.